k8s/0.21.6/0.21.6-local-distroless.yaml

```diff
@@ -998,35 +1006,35 @@ spec:
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: release-name-sentinel
+  name: vsecm-sentinel
   namespace: vsecm-system
   labels:
     helm.sh/chart: sentinel-0.21.6
-    app.kubernetes.io/name: release-name-sentinel
-    app.kubernetes.io/instance: release-name
+    app.kubernetes.io/name: vsecm-sentinel
+    app.kubernetes.io/instance: vsecm
     app.kubernetes.io/part-of: vsecm-system
     app.kubernetes.io/version: "0.21.6"
     app.kubernetes.io/managed-by: Helm
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app.kubernetes.io/name: release-name-sentinel
```

Comment:
the generator script automatically creates release-name instead of vsecm

will file a ticket for that.